pbs/usersync.go

```diff
@@ -77,17 +81,17 @@ func ParseUserSyncMap(cookie *http.Cookie) UserSyncMap {
 // NewSyncMap returns an empty UserSyncMap
 func NewSyncMap() UserSyncMap {
 	return &cookieImpl{
-		UIDs:     make(map[string]string),
-		Birthday: timestamp(),
+		TemporaryUIDs: make(map[string]temporaryUid),
```

Comment:
can you rename UIDs to `legacyUIDs` and keep `UIDs` for the new ones that expire?